lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -263,3 +263,35 @@ function dosomething_rogue_rb_exists_in_rogue($rbid)
 {
   return db_query("SELECT * FROM {dosomething_rogue_reportbacks} rogue_rbs WHERE rbid = :rbid", array(':rbid' => $rbid))->fetchAll();
 }
+
+/**
+ * Checks to see if the given RB data exists in Phoenix and if so stores in Rogue reference table.
+ *
+ * @param array $rogue_reportback
+ * The response from dosomething_rogue_send_reportback_to_rogue().
+ *
+ */
+function dosomething_rogue_check_rbid_and_store_ref($rogue_reportback)
+{
+  $rbid = dosomething_reportback_exists($rogue_reportback['data']['campaign_id'], $rogue_reportback['data']['campaign_run_id'], $rogue_reportback['data']['drupal_id']);
+
+  // Make sure the reportback exists (meaning it got back from Rogue)
+  if ($rbid) {
+    // Store reference to the rb in rogue
+   $reportback = entity_load_unchanged('reportback', [$rbid]);
+    // If a file was added, store references to the relevant Rogue data
+    if (isset($file)) {
```

Comment:
actually, I think this check for a new file can go altogether, because each reference is for an item so we don't want to call this function if there is no new file